app-ios/tutanota/Sources/Contacts/IosMobileContactsFacade.swift

```diff
@@ -150,6 +150,16 @@ class IosMobileContactsFacade: MobileContactsFacade {
 		}
 	}
 
+	private func enumerateContactsInContactStore(
```

Comment:
I would prefer extension for `ContactStore` with something like `enumerateContactsCatching` but that's just a nitpick